jupyter-book/evaluation/evaluation_quiz_m7_04.md

```diff
@@ -11,10 +11,16 @@ What is the default score in scikit-learn when using a classifier?
 +++
 
 ```{admonition} Question
-Which classifier's method should you call to get the predictions required
-by the function `precision_recall_curve`?
```

Comment:
It makes me think that we should be presenting the concept of `predict_proba` and `decision_function` at some points in the course :)